src/components/Search.js

```diff
@@ -3,19 +3,20 @@ import '../styles/Search.css';
 
 export default ({ handleInputChange, inputText, handleClearInput }) => {
   return (
-    <>
-      <input
-        className="search-input"
-        onChange={handleInputChange}
-        value={inputText}
-        type="text"
-        placeholder="search for your item..."
-      />
-      {inputText.length > 0 && (
-        <button className="clear-button" onClick={handleClearInput}>
-          X
-        </button>
-      )}
-    </>
+    <div>
```

Comment:
Curious to know more about why you chose to replace this `React.Fragment` with a `div`. Was it a styling update?

Reply:
I was adding a styling to fix an accessibility error, but then I found another way and forgot to replace the div.